app/controllers/build_estimates_controller.rb

```diff
@@ -20,7 +20,7 @@ def show
     handler = HANDLER_CLASSES[step]
     @form = if handler
               handler.model(session_data)
-            else
+            elsif step == :results
```

Comment:
I have been thinking that this last step doesn't belong here - we've stopped the 'building' phase so maybe this should be a 'create estimate' (in estimates controller). WDYT? Should we have another PR for it?

Reply:
I think that's a very sensible idea, and also that it should be in scope for this ticket, so I'll sort it.